### DISCUSSION TITLE
scopeV2 mRNA版本选择问题

### DISCUSSION THREAD

**Discussion by Li-mengjie** [singleron-RD/CeleScope#179] [Q&A]
老师您好，我使用新格元公司的单细胞建库平台进行单细胞转录组及免疫组库测序，我看到这个软件有很多scope版本，比如scopeV2.2.1，在counts脚本中说这个版本可以自己检测到匹配的版本，但是在我的数据中，自己匹配不到，所以想指定使用某个版本，但是不了解各个版本的区别以及如何根据自己的情况选择，请问老师可以详细解答下吗

**Comment by zhouyiqi91** [comment]
您好，您使用的celescope版本是什么？免疫组库测序是CDR3还是全长？
如果是最新的celescope版本，所有的文库类型都是可以自动检测的。

关于chemistry的含义可以参考
https://github.com/singleron-RD/CeleScope/blob/master/docs/chemistry.md
